### ISSUE TITLE
Raspberry Pi 3B No such file or directory

### ISSUE THREAD

**Issue by Wolfssarah2** [mifi/SimpleInstaBot#116]
Hello, I wanted to run the bot on my pi 3b, I did exactly what you stated in your readme, how to run it on a pi. But i get that weird massage "-bash : ./simpleinstabot: No such file or directory", when I want to start it. (The file is in the directory). Maybe you can help me. Thank you :)

**Comment by mifi** [comment]
hi. maybe you need to `chmod +x simpleinstabot`

**Comment by Wolfssarah2** [comment]
Hello, thanks for the advice, already did, but still the same problem :(

**Comment by mifi** [comment]
Im confused, does the file exist or not? Could you run ls and paste here?

**Comment by Wolfssarah2** [comment]
pi@raspberrypi:~/SimpleInstaBot-linux-armv7l $ ls
chrome_100_percent.pak  libGLESv2.so            resources.pak
chrome_200_percent.pak  LICENSE.electron.txt    simpleinstabot
chrome-sandbox          LICENSES.chromium.html  snapshot_blob.bin
icudtl.dat              locales                 swiftshader
libEGL.so               natives_blob.bin        v8_context_snapshot.bin
libffmpeg.so            resources

**Comment by mifi** [comment]
I really don't know what's going on here

**Comment by Wolfssarah2** [comment]
Maybe here is the problem, but I don't know how to solve it. File gives me:
file simpleinstabot
simpleinstabot: ELF 32-bit LSB shared object, ARM, EABI5 version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-armhf.so.3, for GNU/Linux 3.2.0, BuildID[sha1]=54c9ca38b7ff73d02fe174ce893caffa5756b3cd, stripped

And ldd gives me:

ldd simpleinstabot
        not a dynamic executable

**Comment by NicolasMazzoleni** [comment]
Hi guys,

I've got the exact same issue, so far didn't find a solution. On a raspberry pi 3 freshly reformarted 64 bits

**Comment by NicolasMazzoleni** [comment]
Also, is there another way to execute the program like a .dmg on mac, but for raspbian ? Cheers

**Comment by NicolasMazzoleni** [comment]
Hey guys, found the issue, I was running a 64 bits version of Raspbian, fixed by downgrading to 32 👍

**Comment by c-chytoudis** [comment]
Hello @mifi , is it possible to have a arm64 version for raspberry pi? 
Or to run the armv7 version on arm64 platform?

**Comment by mifi** [comment]
yes i added arm64 now, will be avail in the next release 89f837bc906473cc0ab5f665276777b619ce03bd
